src/lib/components/lines-and-dots/svg/text.svelte

```diff
@@ -48,36 +36,27 @@
     strokeWidth={backdropHeight}
   />
 {/if}
-{#if showIcon && textAnchor === 'start'}
+{#if showIcon}
   <Icon
     name={icon}
-    x={x - offset}
+    x={textAnchor === 'end' ? x - textWidth - backdropHeight : x}
     y={y - 8}
-    class={dark ? 'text-black' : !backdrop ? 'text-primary' : 'text-white'}
+    class={!backdrop ? 'text-primary' : 'text-white'}
   />
 {/if}
 <text
   bind:this={textElement}
   class="cursor-pointer select-none outline-none {category} text-primary"
   class:label
   class:backdrop
-  class:dark
-  x={textX - offset}
+  x={textX}
   {y}
   font-size={fontSize}
   font-weight={fontWeight}
   text-anchor={textAnchor}
 >
   <slot />
 </text>
-{#if showIcon && textAnchor === 'end'}
```

Comment:
What icon was this? I guess what was removed?

Reply:
Not sure why this code existed actually... It was redundant since the icon would show up in the same place (x and y values were the same for both blocks of code). Only difference between this `Icon` and the other one was the ` && textAnchor === 'end'` vs. ` && textAnchor === 'start'` check.

Reply:
👍